docs/application/native/tutorials/process/performance.md

```diff
@@ -175,61 +184,165 @@ To set the Valgrind options for your application:
+
+   - **leak resolution**
+
+        When performing leak checks, this option determines how willing **Memcheck** is to consider different backtraces to be the same for the purposes of merging multiple leaks into a single leak report. When set to **low**, only the first 2 entries need match. When set to **med**, 4 entries have to match. When **high**, all entries need to match.
+
+      For hardcore leak debugging, you probably want to use **--leak-resolution=high** together with **--num-callers=40** or a similar large number. The **--leak-resolution** setting does not affect **Memcheck**'s ability to find leaks. It only changes how the results are presented.
+
+
+   - **freelist size (blocks)**
+
+       When the client application releases memory using free (in C) or delete (C++), that memory is not immediately made available for re-allocation. Instead, it is marked inaccessible and placed in a queue of freed blocks. The purpose is to defer as long as possible the point at which freed-up memory comes back into circulation. This increases the chance that **Memcheck** is able to detect invalid accesses to blocks for some significant period of time after they have been freed.
+
+      This option specifies the maximum total size, in bytes, of the blocks in the queue. The default value is 20 million bytes. Increasing this increases the total amount of memory used by **Memcheck**, but can result in the detection of invalid uses of freed blocks which would otherwise go undetected.
+
+   - **show reachable blocks**
+
+       When disabled, the memory leak detector only shows "definitely lost" and "possibly lost" blocks. When enabled, the leak detector also shows "reachable" and "indirectly lost" blocks. It shows all blocks, except suppressed ones, so **--show-all** would be a better name for it.
+
+   - **allow partial loads**
+
+       This option controls how **Memcheck** handles word-sized, word-aligned loads from addresses for which some bytes are addressable and others are not. When set to **yes**, such loads do not produce an address error. Instead, loaded bytes originating from illegal addresses are marked as uninitialized, and those corresponding to legal addresses are handled in the normal way.When set to **no**, loads from partially invalid addresses are treated the same as loads from completely invalid addresses, an illegal-address error is issued, and the resulting bytes are marked as initialized.
+
+      The code that behaves in this way is in violation of the ISO C/C++ standards, and must be considered broken. If at all possible, such code must be fixed. This option must be used only as a last resort.
+
+   - **undef value errors**
+
+       This option controls whether **Memcheck** reports uses of undefined value errors. If you do not want to see undefined value errors, set this to **no**. It also has the side effect of slightly speedi
```

Comment:
line 280: The separated lines are merged as one line. (When set to)
line 286: Rewrite the last part of this sentence. (~slightly speeding up **Memcheck**.)

Reply:
done